src/components/shared/input/index.tsx

```diff
@@ -33,34 +40,39 @@ const Input = <
 >({
   name,
   placeholder,
-  onFocus,
+  onFocus = () => null,
```

Comment:
this default is not necessary, undefined is ok to not execute any onFocus when no prop is passed
```suggestion
  onFocus,
```

Reply:
Done.